README.md

```diff
@@ -49,6 +49,20 @@ of your `~/.docker/config.json` file to be:
 }
 ```
 
+This will cause `docker` to use `docker-credential-ecr-login` for all remote registries.
+
+To use this helper for a specific ECR registry, use a `credHelpers` section with the name of your ECR registry:
```

Comment:
To use this credential helper for a specific ECR registry, create a `credHelpers` section with the URI of your ECR registry: